Workflow Commons - Task Assignment

ozgeMendix opened the pull request:
This is still a draft.

ozgeMendix committed:
Workflow Commons - Task assignment functionality

PaulJongenMx commented on content/en/docs/appstore/platform-supported-content/modules/workflow-commons.md:
```diff
@@ -59,6 +59,7 @@ You can find the following pages in Workflow Commons:
 * **TaskInbox** – This page contains a list of all tasks that a user can interact with. **My open tasks** shows the tasks assigned to current users, **All open tasks** is a list of tasks they could pick up and **Unassigned tasks** shows all unassigned tasks.
 * **WorkflowAdminCenter** – A navigational page for workflow administrators. From here, a workflow administrator can go the **Workflow Dashboard**, which gives them general statistics of workflows. Workflow administrators also gain access to **Workflow management**, where they can see all the instances of specific workflows and make changes to their data or even abort workflows.
 * **WorkflowDashboard** – This page gives you workflow/task based metrics and direct access to all *Workflow* and *WorkflowUserTask* data. When you want to use the Workflow Dashboard as your only admin go-to page, add this to the navigation instead of the **WorkflowAdminCenter**. 
+* **TaskAssignment** – This page provides the workflow administrator the ability to manage the assignment and targeting of user tasks based on a selected user. On the page there are two tabs, first is the **Assigned tasks** shows a list of tasks assigned to the selected user, possible actions are reassigning and unassigning the corresponding task(s). Second, **Targeted tasks** shows a list of tas ...
```
after this line "...on a selected user." This is typically done when a user is leaving the organisation or a group/role, where all the tasks a selected user is working on need to be assigned to another user or unassigned from the departing user. 
Rewrite: "On the page, first select the user whose tasks should be listed. There are two tabs, ..."

PaulJongenMx reviewed (commented):
added 1 comment

ozgeMendix committed:
Workflow Commons - Task assignment functionality

ozgeMendix committed:
Workflow Commons - Task assignment functionality

ozgeMendix commented:
This is still a draft. Please, don't merge it, yet. I will be in PTO from Nov 4th - 25th. When the module is ready to be released, someone from my team contact you. Thanks!!

ConnorLand committed:
language and formatting edit

Made writing clearer, broke tabs out into bullets, and deleted "two tabs" to make the doc more future proof.
